docs/pt/docs/tutorial/middleware.md

```diff
@@ -0,0 +1,61 @@
+# Middleware
+
+Você pode adicionar middlewares nas suas aplicações **FastAPI**.
+
+Um "middleware" é uma função que trabalha com todas as **requisições (requests)** antes delas serem processadas por qualquer *operação de caminho*. E também com todas as **respostas (responses)** antes delas serem retornadas.
+
+* Ele pega cada **requisição** que chega para sua aplicação.
+* Depois, ele pode fazer algo com essa **requisição** ou rodar qualquer código necessário.
+* Depois, ele passa a **requisição** para ser processada pelo resto da aplicação (por alguma *operação de caminho*).
```

Comment:
```suggestion
* Depois, a **requisição** é liberada para ser processada pela aplicação (por alguma *operação de caminho*).
```
The `(por alguma *operação de caminho*)` sounds ambiguous to me, because, when I'm reading the docs I'm thinking this means to a Router middleware, if this is what is happening we need to think on how to explain (for a 10 years old) what does it mean.